uniffi/src/bindings/python/templates/RustBufferHelper.py

```diff
@@ -44,7 +44,14 @@ def getInt(self):
         return self._unpack_from(4, ">i")
 
     def putInt(self, v):
-        self._pack_into(8, ">i", v)
+        self._pack_into(4, ">i", v)
```

Comment:
Was not sure if this is okay! I have very little python knowledge soooo

Reply:
This looks reasonable to me 👍